src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/PublicAPI/PublicAPI.Shipped.txt

```diff
@@ -201,3 +201,13 @@ Microsoft.TestPlatform.VsTestConsole.TranslationLayer.ConsoleParameters.Environm
 Microsoft.TestPlatform.VsTestConsole.TranslationLayer.ConsoleParameters.EnvironmentVariables.set -> void
 Microsoft.TestPlatform.VsTestConsole.TranslationLayer.ConsoleParameters.InheritEnvironmentVariables.get -> bool
 Microsoft.TestPlatform.VsTestConsole.TranslationLayer.ConsoleParameters.InheritEnvironmentVariables.set -> void
+static Microsoft.VisualStudio.TestPlatform.VsTestConsole.TranslationLayer.Resources.Resources.CannotFindConsoleRunner.get -> string
+static Microsoft.VisualStudio.TestPlatform.VsTestConsole.TranslationLayer.Resources.Resources.ProcessStartWin32Failure.get -> string
```

Comment:
Any reason to make these 2 APIs `public` instead of `internal`? We probably can't change the others but if that's not too late for this one I would vote for adding a new resource file with accessibility set to `internal` to avoid adding more surface to support.